.gitignore

```diff
@@ -4,3 +4,8 @@ jspm_packages
 
 # Serverless directories
 .serverless
+envvars.sh
```

Comment:
How about putting these files into ./private and having just one entry here for `.private`? That way it works for anyone, regardless of the filenames they choose to use.